Xamarin.PropertyEditing.Mac/Controls/EnumEditorControl.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Collections;
+using System.ComponentModel;
+using System.Diagnostics;
+using AppKit;
+using CoreGraphics;
+using Foundation;
+using Xamarin.PropertyEditing.ViewModels;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	internal class EnumEditorControl : PropertyEditorControl
+	{
+		NSComboBox ComboBoxEditor;
+
+		dynamic EnumEditorViewModel => ViewModel;
```

Comment:
Let's not bring the DLR into this. We can make `EnumEditorControl` generic and initialize it with `MakeGenericType` and `Activator`.